mmcv/runner/utils.py

```diff
@@ -13,7 +14,18 @@
 
 
 def get_host_info():
-    return f'{getuser()}@{gethostname()}'
+    """Get hostname and username.
+
+    Return empty string if exception raised, e.g. `getuser()` will lead to
+    error in docker container
```

Comment:
Maybe we can also mount `/etc/passwd` from our host when we run the docker with the following command to fix the exception. More details at https://stackoverflow.com/questions/48527958/docker-error-getting-username-from-password-database

```bash
docker run -v /etc/passwd:/etc/passwd ...
```

Reply:
> Maybe we can also mount `/etc/passwd` from our host when we run the docker with the following command to fix the exception. More details at https://stackoverflow.com/questions/48527958/docker-error-getting-username-from-password-database
> 
> ```shell
> docker run -v /etc/passwd:/etc/passwd ...
> ```

This should be work, but from the user's point of view, it is even more inconvenient